### DISCUSSION TITLE
Get current authentication strategy

### DISCUSSION THREAD

**Discussion by teawithfruit** [ory/kratos#2808] [Q&A]
How do you get deeper information about the current authentication strategy of a running session? 
If I call the /whoami route I get no deeper information about that.
I would like to know which OIDC provider is used if the user is authenticated with OIDC. 
Until now there is just the information if OIDC or email/password is used, if I'm right.

**Comment by aeneasr** [comment]
Hi, you see this in formation in the `authentication_methods` field of the session!
